linkingFuncs.py

```diff
@@ -173,9 +209,12 @@ def add_watchlist(username, tik):
         return { 'Action': False}
```

Comment:
More descriptive returned JSON could be helpful for debugging but not absolutely necessary